subgraph-kovan-staging.yaml

```diff
@@ -205,4 +207,42 @@ templates:
       eventHandlers:
         - event: Transfer(indexed address,indexed address,uint256)
           handler: handleERC20Transfer
-      file: ./src/mappings/ERC20.ts
\ No newline at end of file
+      file: ./src/mappings/ERC20.ts
+  - kind: ethereum/contract
+    name: Tranche
+    network: kovan
+    source:
+      abi: Tranche
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      entities:
+        - TokenBalance
+      abis:
+        - name: Tranche
+          file: ./abis/Tranche.json
+        - name: ERC20
+          file: ./abis/ERC20.json
+      callHandlers:
+        - function: supplyOrder(address,uint256)
+          handler: handleSupplyOrder
```

Comment:
We don't need to handle redeem because upon placing a redeem order you move the ERC20 tokens into the Tranche thus they are already not counted anymore when we look at token balances.